src/themes/dark.scss

```diff
@@ -1,6 +1,13 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-theme-dark {
   --wxc-meeting-background: #{$wxc-gradation-webex-dark-primary};
   --wxc-text-color: #{$wxc-white-alpha-100};
+
+  //Banner
+  --wxc-banner-background: linear-gradient(2.25deg, rgba(0, 0, 0, 0) 1.9%, rgba(0, 0, 0, 0.3) 50.23%, rgba(0, 0, 0, 0.8) 98.11%);
```

Comment:
this gradient should already be defined in _colors.scss

Reply:
it's not, but I will add it